.github/workflows/gitlab.yml

```diff
@@ -31,4 +31,4 @@ jobs:
           project-id: ${{ secrets.PROJECT_ID }}
           schedule: ${{ github.event_name == 'schedule' }}
           cancel-outdated-pipelines: ${{ github.ref_name != 'main' }}
-          triggered-ref: v1.1.12
+          triggered-ref: LLT-5252_update_gitlab_url
```

Comment:
I guess you will update it to `v1.1.13` after merging?

Reply:
correct, thanks for leaving a reminder :)